# Problem 5.53 — Relativistic Momentum: Helium Nucleus

## Problem Statement

Find the **relativistic momentum** of a helium nucleus having a mass of $m = 6.68 \times 10^{-27}$ kg that is moving at $v = 0.200c$.

---

## Background

In classical mechanics, momentum is simply $p = mu$. But at relativistic speeds, this is not conserved across all inertial frames. The correct relativistic momentum is:

$$\vec{p} = \gamma m \vec{u} = \frac{m\vec{u}}{\sqrt{1 - u^2/c^2}}$$

where:
- $m$ is the **rest mass**
- $\vec{u}$ is the velocity of the object
- $\gamma$ is the Lorentz factor

---

## Given

| Quantity | Value |
|----------|-------|
| Rest mass of helium nucleus | $m = 6.68 \times 10^{-27}$ kg |
| Speed | $v = 0.200c$ |
| Speed of light | $c = 3.00 \times 10^8$ m/s |

In [1]:
import math

# ── Given ─────────────────────────────────────────────────────────────
c    = 3.00e8             # speed of light, m/s
m    = 6.68e-27           # rest mass of He nucleus, kg
v    = 0.200 * c          # speed, m/s

# ── Step 1: Compute β ─────────────────────────────────────────────────
beta = v / c
print(f"Step 1 — β = v/c = {beta:.4f}")

# ── Step 2: Compute γ ─────────────────────────────────────────────────
gamma = 1 / math.sqrt(1 - beta**2)
print(f"Step 2 — γ = 1/√(1−β²)")
print(f"         γ = 1/√(1 − {beta**2:.6f})")
print(f"         γ = 1/√({1-beta**2:.6f})")
print(f"         γ = {gamma:.6f}")

# ── Step 3: Relativistic momentum ─────────────────────────────────────
p_rel = gamma * m * v
print(f"\nStep 3 — p = γmv")
print(f"         p = {gamma:.6f} × {m:.3e} kg × {v:.4e} m/s")
print(f"         p = {p_rel:.4e} kg·m/s")

# ── Compare with classical momentum ───────────────────────────────────
p_class = m * v
print(f"\nClassical momentum (p = mv):")
print(f"         p_class = {p_class:.4e} kg·m/s")

fractional_diff = (p_rel - p_class) / p_class * 100
print(f"\nFractional difference: {fractional_diff:.4f}%")
print(f"  (Relativistic momentum is {fractional_diff:.4f}% larger than classical)")

print(f"\n{'='*55}")
print(f"ANSWER: p_rel = {p_rel:.4e} kg·m/s")

Step 1 — β = v/c = 0.2000
Step 2 — γ = 1/√(1−β²)
         γ = 1/√(1 − 0.040000)
         γ = 1/√(0.960000)
         γ = 1.020621

Step 3 — p = γmv
         p = 1.020621 × 6.680e-27 kg × 6.0000e+07 m/s
         p = 4.0906e-19 kg·m/s

Classical momentum (p = mv):
         p_class = 4.0080e-19 kg·m/s

Fractional difference: 2.0621%
  (Relativistic momentum is 2.0621% larger than classical)

ANSWER: p_rel = 4.0906e-19 kg·m/s


## Step-by-Step Solution

### 1. Compute β and γ

$$\beta = \frac{v}{c} = \frac{0.200c}{c} = 0.200$$

$$\gamma = \frac{1}{\sqrt{1 - (0.200)^2}} = \frac{1}{\sqrt{1 - 0.0400}} = \frac{1}{\sqrt{0.9600}} = \frac{1}{0.9798} \approx 1.0206$$

### 2. Compute relativistic momentum

$$p = \gamma m v = 1.0206 \times (6.68 \times 10^{-27}\,\text{kg}) \times (0.200 \times 3.00 \times 10^8\,\text{m/s})$$

$$\boxed{p \approx 4.09 \times 10^{-19}\text{ kg·m/s}}$$

### 3. Compare with classical

$$p_{\text{class}} = mv = (6.68 \times 10^{-27})(6.00 \times 10^7) = 4.01 \times 10^{-19}\text{ kg·m/s}$$

The relativistic momentum is about **2.06% larger** than the classical value.

## Summary

$$\boxed{p = \gamma m v = 1.0206 \times (6.68 \times 10^{-27}) \times (6.00 \times 10^7) \approx 4.09 \times 10^{-19}\,\text{kg·m/s}}$$

At $v = 0.200c$, γ ≈ 1.021, so the relativistic momentum is about 2% larger than the classical value.